htdocs/comm/remx.php

```diff
@@ -1060,3 +1063,13 @@
 // End of page
 llxFooter();
 $db->close();
+?>
+<script>
+    jQuery(document).ready(function () {
+	jQuery("#discount_amount_ttc").keyup(function () {
+	var tva_tx = 1+(jQuery("#tva_tx").val()/100);
+	var value = Math.round10(jQuery(this).val()/1.18,-8);
```

Comment:
What is 1.18 and -8 ?
No hard coded values are allowed.

Reply:
Hi.
I forgot it, 1.18 is tva_tx.
-8 is th precition, I think it's 'MS'?

Reply:
MS is better than -8, because it takes into account the accuracy defined into user by setup./
A good way to deal with this vat calculation is:
when you enter data into price without tax to completely empty the input field for price with tax and if we enter a value into field with tax, we empty in javascript the value without tax.
And we calculate the price with or without by the PHP after the submit, so we can use the method get_default_tva() to kwon the vat to use and before making the insert or create into database.

Reply:
That is all right. Thank you for your help.